.github/workflows/release.yml

```diff
@@ -118,6 +118,63 @@ jobs:
           path: build/release/nvim-macos.tar.gz
           retention-days: 1
 
+  macOS-universal:
+    runs-on: macos-11
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Install brew packages
+        run: |
+          brew update --quiet
+          brew install automake ninja
+      - if: github.event_name == 'push' || (github.event_name == 'workflow_dispatch' && github.event.inputs.tag_name != 'nightly')
+        run: printf 'NVIM_BUILD_TYPE=Release\n' >> $GITHUB_ENV
+      - if: github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.event.inputs.tag_name == 'nightly')
+        run: printf 'NVIM_BUILD_TYPE=RelWithDebInfo\n' >> $GITHUB_ENV
+      - name: Provision universal `libintl`
+        run: |
+          bottle_tag="arm64_big_sur"
+          brew fetch --bottle-tag="$bottle_tag" gettext
+          cd "$(mktemp -d)"
+          tar xf "$(brew --cache)"/**/*gettext*${bottle_tag}*.tar.gz
+          gettext_prefix="$(brew --prefix gettext)"
+          broken_install_name="$(otool -D gettext/*/lib/libintl.dylib | sed 1d)"
+          fixed_install_name="$(otool -D "${gettext_prefix}/lib/libintl.dylib" | sed 1d)"
+          libintl_basename="$(basename "$fixed_install_name")"
+          install_name_tool -change "$broken_install_name" "$fixed_install_name" gettext/*/lib/"$libintl_basename"
+          lipo gettext/*/lib/"$libintl_basename" \
+               "${gettext_prefix}/lib/${libintl_basename}" \
+               -create -output "$libintl_basename"
+          mv -f "$libintl_basename" "${gettext_prefix}/lib/${libintl_basename}"
+      - name: Build release
+        run: |
+          CMAKE_EXTRA_FLAGS="-DCMAKE_INSTALL_PREFIX:PATH= -DCMAKE_OSX_DEPLOYMENT_TARGET=11 -DCMAKE_OSX_ARCHITECTURES=\\\"arm64;x86_64\\\""
```

Comment:
Tried testing this in my fork. Unfortunately, `make` chokes on the `;` inside `CMAKE_OSX_ARCHITECTURES`. Not sure how to get around that.

Reply:
Try `\;`

(in cmake, `;` is how a "list" is internally encoded, but `\;` should work around this)

Reply:
Yes, that seems to work. Thanks.